lib/livebook_web/live/session_live/indicators_component.ex

```diff
@@ -85,6 +85,9 @@ defmodule LivebookWeb.SessionLive.IndicatorsComponent do
             <span>Presentation</span>
           </button>
         </.menu_item>
+        <.menu_item>
+        <.custom_view_button session_id={@session_id} ></.custom_view_button>
```

Comment:
This button is using a different styling than the ones above. Can you please make sure they match? I don't think we need a custom function component here either, I think we can just inline the `button` here. :)